psychopy/app/builder/builder.py

```diff
@@ -558,9 +559,8 @@ def __init__(self, frame, id=-1):
     def getCurrentRoutine(self):
         routinePage = self.getCurrentPage()
         if routinePage:
-            return routinePage.routine
-        else:  # no routine page
```

Comment:
I actually prefer this explicit statement. Matter of taste? What do the other devs think? @peircej?